### PR TITLE
Web-app and Backend

### PR DESCRIPTION
**Pull request by julianfl0w**
This PR includes a working configuration for several backend security services (Keycloak and OpenTDF), fairly parameterized in env.py. 

run.py will bring up the entire backend and web-app, and has been set to do so as a Github Action at push time.

### CODE REVIEW

File: diagrams/architecture.dot [hunk 1]
```diff
@@ -0,0 +1,17 @@
+digraph Architecture {
```
Comment (arkavo-com) [inline diagrams/architecture.dot:1]: mermaid diagram would get rendered then no need for the png

File: nginx/accounts/acme-v02.api.letsencrypt.org/directory/da624814afaf54756796d70d805a1161/private_key.json
Comment (arkavo-com) [inline nginx/accounts/acme-v02.api.letsencrypt.org/directory/da624814afaf54756796d70d805a1161/private_key.json:1]: add "mock" or "test" to the filename.

**Review by arkavo-com — APPROVED** [review]

File: diagrams/architecture.dot (hunk 1 above)
Comment (julianfl0w) [reply diagrams/architecture.dot:1]: ive added Mermaid diagram to the readme. it is convenient, but may not look as good

File: nginx/accounts/acme-v02.api.letsencrypt.org/directory/da624814afaf54756796d70d805a1161/private_key.json
Comment (julianfl0w) [reply nginx/accounts/acme-v02.api.letsencrypt.org/directory/da624814afaf54756796d70d805a1161/private_key.json:1]: i've removed this auto-generated folder entirely

**Pull request merged**